spec/flows/under_eighteen_flow_spec.rb

```diff
@@ -1,27 +1,30 @@
 require "rails_helper"
 
 RSpec.describe "Under 18 flow", type: :feature do
-  it "shows additional questions form u18 controlled checks if flag is enabled" do
+  let(:clr_text) { "Is the work controlled legal representation" }
+
+  before do
     start_assessment
     fill_in_client_age_screen(choice: "Under 18")
+  end
+
+  it "shows additional questions form u18 controlled checks if flag is enabled" do
```

Comment:
would you mind quickly taking out the "if flag is enabled" as we got rid of that flag ages ago